semanticdb/metac/src/main/scala/scala/meta/cli/Metac.scala

```diff
@@ -20,7 +21,8 @@ object Metac {
   }
 
   private def process(args: Array[String], reporter: Reporter): Int = {
-    Settings.parse(args.toList, reporter) match {
+    val expandedArgs = Args.expand(args)
```

Comment:
Why expand the args before calling `Settings.parse`? For me, this logic is the responsibility of `Settings.parse`. I don't have a strong opinion however as long as it's consistent.

Reply:
What I had in mind here is that the `@args` expansion is only relevant to CLI invocations, since it's only the CLI invocations that are limited by the max size of a shell command. As you said, this is consistent - both within Scalameta and within Rsc.